{"ms.devlang":"csharp","author":"erickson-doug","ms.author":"douge","manager":"douge","breadcrumb_path":"breadcrumb/toc.json","ms.topic":"managed-reference","ms.service":"service-fabric","ms.date":"11/16/2016","search.ms_sitename":"Docs","search.ms_docsetname":"azuredotnet","version":0,"locale":"en-us","site_name":"Docs","search.ms_product":"Azure","depot_name":"Azure.azuredotnet","ref_skeleton_gitcommit":"https://github.com/Azure/azure-docs-sdk-dotnet/blob/8ece26063f2f27414f0bad873159ee6212f3448d/docs-ref-manual/Data/ServiceFabric/System.Fabric.Testability.Scenario.ChaosTestScenario.yml","original_ref_skeleton_git_url":"https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/docs-ref-manual/Data/ServiceFabric/System.Fabric.Testability.Scenario.ChaosTestScenario.yml","open_to_public_contributors":false,"api_name":["System.Fabric.Testability.Scenario.ChaosTestScenario","System.Fabric.Testability.Scenario.ChaosTestScenario..ctor","System.Fabric.Testability.Scenario.ChaosTestScenario.OnExecuteAsync","System.Fabric.Testability.Scenario.ChaosTestScenario.OnDispose","System.Fabric.Testability.Scenario.ChaosTestScenario.ValidateScenarioAtExitAsync"],"api_location":["System.Fabric.dll"],"topic_type":["apiref"],"api_type":["Assembly"],"f1_keywords":["System.Fabric.Testability.Scenario.ChaosTestScenario","System::Fabric::Testability::Scenario::ChaosTestScenario","System.Fabric.Testability.Scenario.ChaosTestScenario.#ctor","System::Fabric::Testability::Scenario::ChaosTestScenario::#ctor","System.Fabric.Testability.Scenario.ChaosTestScenario.OnExecuteAsync","System::Fabric::Testability::Scenario::ChaosTestScenario::OnExecuteAsync","System.Fabric.Testability.Scenario.ChaosTestScenario.OnDispose","System::Fabric::Testability::Scenario::ChaosTestScenario::OnDispose","System.Fabric.Testability.Scenario.ChaosTestScenario.ValidateScenarioAtExitAsync","System::Fabric::Testability::Scenario::ChaosTestScenario::ValidateScenarioAtExitAsync"],"dev_langs":["csharp","vb"],"updated_at":"2016-12-19 07:26 AM","document_id":"48ed783c-061e-2f2f-03f6-26d70ce39a4b","layout":"Reference","pagetype":"Reference","title":"ChaosTestScenario class | Microsoft Docs","description":"The ChaosTestScenario is a long running scenario which keeps inducing failover and faults into the cluster up until the TimetoRun has expired.\n","toc_asset_id":"toc.json","toc_rel":"toc.json","content_git_url":"","source_url":"https://msazure.visualstudio.com/DefaultCollection/One/_git/WindowsFabric?path=src/prod/src/managed/Api/src/System/Fabric/Testability/Scenario/ChaosTestScenario.cs&version=GBdevelop&line=27","ms.assetid":"System.Fabric.Testability.Scenario.ChaosTestScenario","canonical_url":"https://docs.microsoft.com/en-us/dotnet/api/system.fabric.testability.scenario.chaostestscenario","is_dynamic_rendering":false,"content_type":"text/html","publish_version":"{\"opbuild.templates.docs\":\"1.28.1\",\"commit_id\":\"b04ff3a8f954034adf9b3f9688d5b3d06978b7e8\",\"branch\":\"master\",\"opbuild.scripts\":\"1.28.1\",\"opbuild.htmltopdftool\":\"1.28.1\",\"opbuild.gitcontributorinformationresolver\":\"1.28.1\",\"opbuild_service\":\"1.28.303.0\",\"docfx.console\":\"2.13.0\",\"opbuild.resolvedependencyconsole\":\"1.28.1\",\"opbuild.schemavalidationconsole\":\"1.28.1\",\"opbuild.redirectionresolver\":\"1.28.1\",\"opbuild.templates.intellisense\":\"1.28.1\",\"dependencies\":[{\"branch\":\"master\",\"path\":\"_themes\",\"url\":\"https://github.com/Microsoft/templates.docs.msft\",\"commit_id\":\"7d3ef1a51554bb393cfbe59160c498316c9c2dc3\",\"opbuild.templates.common\":\"1.28.1\"}]}"}